src/docker/install.sh

```diff
@@ -5,7 +5,7 @@ set -e
 ORCH_DIR=$1
 cd $ORCH_DIR
 
-sbt -batch compile
+sbt -batch -d clean reload update compile
```

Comment:
this _may_ be the source of some build wonkiness - we weren't cleaning before building. There may have been stale caches with incorrect info in them. sbt _should_ handle that but it also feels correct to have a clean step here.

The `-d` might be overkill but was helpful during this PR, so I'm leaving it in.

Reply:
Thank you!!